src/api-engine/api/routes/chaincode/views.py

```diff
@@ -145,7 +145,7 @@ def package(self, request):
                 peer_channel_cli = PeerChainCode("v2.2.0", **envs)
                 res = peer_channel_cli.lifecycle_package(name, version, chaincode_path, language)
                 os.system("rm -rf {}/*".format(file_path))
-                os.system("mv {}.tar.gz {}".format(name, file_path))
+                os.system("mv {}.tar.gz {}".format(name, secure_file_path(file_path)))
```

Comment:
There is a couple of issues.
1. secure_file_path isn't  imported
2. file_path is declared internally, I don't think we need to make this change.

Reply:
Isn't that necessary on this function?

Reply:
Please address yuanmao's comments before the merging.